hubverse_annotator/ui.py

```diff
@@ -426,8 +428,22 @@ def plotting_ui(
     title = f"{loc_abbr}: {selected_target}, {selected_ref_date}"
     chart = (
         layer.interactive()
-        .properties(title=alt.TitleParams(text=title, anchor="middle"))
-        .facet(row=alt.Row("model_id:N"), columns=1)
+        .facet(
+            row=alt.Row(
+                "model_id:N",
+                header=alt.Header(
+                    labelColor="black",
+                    labelFontSize=Y_LABEL_FONT_SIZE,
+                    title=None,
+                ),
+            ),
+            columns=1,
+        )
+        .properties(
+            title=alt.TitleParams(
+                text=title, fontSize=18, fontWeight="bold", anchor="middle"
```

Comment:
How are you deciding which variables are configured at the top (`labelFontSize`) and which are not (`fontSize`)?

Reply:
I'm asking about your decisions to have `labelFontSize=Y_LABEL_FONT_SIZE` and `fontSize=18`. They both change font sizes, but one is a constant defined at the top of the file, and the other is defined inline. Do you have a reason for the different approaches?